test/safe-mode-check.js

```diff
@@ -7,27 +7,20 @@ import {JsonLdDocumentLoader} from 'jsonld-document-loader';
 
 const jdl = new JsonLdDocumentLoader();
 
-// Loop all local context files and add them as static contexts at the base URL
-const paths = (await glob([`${contextsDir}/**/*.json`],
-  {stat: true, withFileTypes: true}))
-  .map(path => path.fullpath());
+const contextUrls = new Set();
```

Comment:
Add brief documentation
```suggestion
// collection to track all loaded contexts
const contextUrls = new Set();
```